pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -37,15 +42,16 @@ type Monitor struct {
 
 	// lastStatus keeps track of OSDs status
 	// key - OSD id; value: time of the status change.
-	lastStatus map[int]time.Time
+	lastDownStatus map[int]time.Time
+	lastOutStatus  map[int]time.Time
```

Comment:
Do we need to track the out status? If an OSD is marked as `out`, we don't take any action until it is also `safe-to-destroy`, right? If it's out and safe to destroy, we can just remove it at that point rather than tracking in this map.